Tests/js/views/fields/assets/ez-richtext-editview-tests.js

```diff
@@ -821,11 +740,17 @@ YUI.add('ez-richtext-editview-tests', function (Y) {
     Y.Test.Runner.add(getFieldTest);
     Y.Test.Runner.add(editorTest);
     Y.Test.Runner.add(focusModeTest);
-    Y.Test.Runner.add(actionBarTest);
-    Y.Test.Runner.add(removeYuiIdTest);
-    Y.Test.Runner.add(destructorTest);
     Y.Test.Runner.add(appendToolbarConfigTest);
     Y.Test.Runner.add(registerTest);
     Y.Test.Runner.add(editorFocusHandlingTest);
     Y.Test.Runner.add(eventForwardTest);
-}, '', {requires: ['test', 'base', 'view', 'node-event-simulate', 'fake-toolbarconfig', 'editviewregister-tests', 'ez-richtext-editview']});
```

Comment:
ok the line is bit long, but if you fix that, that should be correct CS wise, ie:

```js
}, '', {
    requires: [
        'test',
        'base',
        'view',
        'node-event-simulate',
        'fake-toolbarconfig',
        'editviewregister-tests',
        'ez-richtext-editview',
    ]
});